crypto/dsa/dsa_ossl.c

```diff
@@ -259,7 +260,7 @@ static int dsa_sign_setup(DSA *dsa, BN_CTX *ctx_in,
         goto err;
 
     /* Compute  part of 's = inv(k) (m + xr) mod q' */
-    if ((kinv = BN_mod_inverse(NULL, k, dsa->q, ctx)) == NULL)
+    if ((kinv = dsa_inverse_mod(k, dsa->q, ctx)) == NULL)
```

Comment:
Just a curious question from a consistency addict: why the reordering from `mod_inverse` to `inverse_mod`?